nikola/utils.py

```diff
@@ -1272,13 +1286,66 @@ def publish_extensions(self, handler):
             })
             handler.endElement("atom:link")
 
+        if self.itunes_author:
```

Comment:
You could duplicate less code, eg. with `getattr` or some iTunes data dictionary.